### ISSUE TITLE
blank page

### ISSUE THREAD

**Issue by nephilim75** [pstimpel/otpweb#1]
Hello

trying this solution but finally I end up on a blank page. NGINX error log says:
```
thrown in /var/www/otpweb-master/index.php on line 135" while reading response header from upstream, client: 192.168.178.2, server: otp.mydomain.de, request: "GET / HTTP/1.1", upstream: "fastcgi://unix:/run/php>
2023/07/24 20:40:11 [error] 13164#13164: *9 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught Error: Class "Session" not found in /var/www/otpweb-master/index.php:135
```

System:
OS: debian 12
PHP: 8.2
nginx: nginx/1.22.1
Postgres 15

My php configuration looks like this:
```
        location ~ \.php$ {
                fastcgi_pass unix:/run/php/php8.2-fpm.sock;
                fastcgi_index index.php;
                include fastcgi_params;
                fastcgi_param SCRIPT_FILENAME /var/www/otpweb-master/$fastcgi_script_name;
        }
```

Any idea how I can fix this? The error log points to index.php line 135, but maybe my PHP config is not working properly?

Kind regards
//neph

**Comment by pstimpel** [comment]
Wow, sorry for the late reply. I did not receive any notification from github.

Can you confirm the session.php file in the subfolder init is existing?

Other question: did this happen before any setup in your browser, or after the setup?

**Comment by pstimpel** [comment]
cannot reproduce, closed

**Comment by pstimpel** [comment]
solved in 1.0.8
